hal/lib/athena/FRCDriverStation.cpp

```diff
@@ -27,6 +28,7 @@ struct HAL_JoystickAxesInt {
 static priority_mutex msgMutex;
 static priority_condition_variable newDSDataAvailableCond;
 static priority_mutex newDSDataAvailableMutex;
+static std::atomic_int newDSDataAvailableCounter {0};
```

Comment:
Since you have to grab a lock anyway (at least when reading), maybe use just an int instead of an atomic int? yes, you'll have to grab a lock when writing it, but I think that's okay.